Dockerfile

```diff
@@ -10,40 +10,43 @@ ADD rebuild_drivers /tmp
 RUN yum-config-manager --save --setopt=doca.sslverify=0 doca
 RUN yum-config-manager --save --setopt=doca.gpgcheck=0 doca
 RUN yum-config-manager --dump doca
 
 
-RUN for app in wget util-linux netplan openssh-server iproute which git selinux-policy-devel diffutils file procps-ng patch rpm-build kernel kernel-devel kernel-headers libreswan python3-devel python3-test python3-Cython efibootmgr efivar grub2 grub2-efi grub2-efi-unsigned shim-unsigned-aarch64 lvm2 popt-devel bc flex bison lm_sensors ninja-build meson cryptsetup rasdaemon pciutils-devel python3-sphinx python3-six kexec-tools jq dbus libgomp iana-etc libgomp-devel libgcc-devel libgcc-atomic libmpc binutils iptables glibc-devel gcc tcl-devel automake libmnl autoconf tcl libnl3-devel openssl-devel libstdc++-devel binutils-devel libnl3 libdb-devel make libmnl-devel iptables-devel lsof desktop-file-utils doxygen cmake cmake3 libcap-ng-devel systemd-devel ncurses-devel net-tools sudo libpcap libnuma unbound vim iputils; do yum install -y $app || true; done
+# RUN for app in wget util-linux netplan openssh-server iproute which git selinux-policy-devel diffutils file procps-ng patch rpm-build kernel kernel-devel kernel-headers libreswan python3-devel python3-test python3-Cython efibootmgr efivar grub2 grub2-efi grub2-efi-unsigned shim-unsigned-aarch64 lvm2 popt-devel bc flex bison lm_sensors ninja-build meson cryptsetup rasdaemon pciutils-devel python3-sphinx python3-six kexec-tools jq dbus libgomp iana-etc libgomp-devel libgcc-devel libgcc-atomic libmpc binutils iptables glibc-devel gcc tcl-devel automake libmnl autoconf tcl libnl3-devel openssl-devel libstdc++-devel binutils-devel libnl3 libdb-devel make libmnl-devel iptables-devel lsof desktop-file-utils doxygen cmake cmake3 libcap-ng-devel systemd-devel ncurses-devel net-tools sudo libpcap libnuma unbound vim iputils; do yum install -y $app || true; done
+
+RUN yum install -y wget kmod util-linux sudo netplan hostname openssh-server iproute which git selinux-policy-devel diffutils file procps-ng patch rpm-build kernel kernel-devel kernel-headers python3 python3-devel python3-libs python3-test python3-pyelftools efibootmgr efivar grub2 grub2-efi grub2-efi-unsigned shim-unsigned-aarch64 lvm2 popt-devel bc flex bison lm_sensors ninja-build meson cryptsetup pciutils-devel python3-sphinx python3-six kexec-tools jq dbus libgomp iana-etc libgomp-devel libgcc-devel libgcc-atomic libmpc binutils libsepol-devel iptables glibc-devel gcc tcl-devel automake libmnl autoconf tcl libnl3-devel openssl-devel libstdc++-devel binutils-devel libselinux-devel libnl3 libdb-devel make libmnl-devel iptables-devel lsof glibc numactl-devel ncurses-devel systemd-devel groff libgudev libgudev-devel lsb-release popt-devel pkg-config python3-twisted libpcap unbound python3-zope-interface graphviz less iputils tcpdump sysstat qemu-kvm libvirt libguestfs-tools libreswan ipmitool nvme-cli coreutils rsyslog
```

Comment:
Minor note: for easier reading , these longs lists of packages (here and below on line 30) may be benefit from being split across lines and sorted. That will certainly make it easier for reviewers to understand what's changed. For example:

```
RUN yum install -y \
    coreutils \
    meson \
    wget
```

Reply:
(I also like that we're now installing them all in one go instead of separate invocations of `yum install`; that should hopefully also be a little faster.)